docs/content/mixins/exec.md

```diff
@@ -33,4 +33,12 @@ install:
     command: bash
     arguments:
     - ./install-world.sh
-```
\ No newline at end of file
+```
+
+### FAQ
+
+#### How do I use pipes?
+
+If you have a command that pipes, place the command in a script file and then
+use the exec mixin to invoke that script passing in parameters any parameters 
```

Comment:
There's an extra `parameters` here, it seems.

Reply:
Fixed